bazel/repository.bzl

```diff
@@ -0,0 +1,80 @@
+load("@bazel_tools//tools/build_defs/repo:utils.bzl", "maybe")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+def opentelemetry_cpp_deps():
+    """Loads dependencies need to compile the opentelemetry-cpp library."""
+
+    # Google Benchmark library.
+    # Only needed for benchmarks, not to build the OpenTelemetry library.
+    maybe(
+        native.local_repository,
+        name = "com_github_google_benchmark",
+        path = "third_party/benchmark",
+    )
+
+    # GoogleTest framework.
+    # Only needed for tests, not to build the OpenTelemetry library.
+    maybe(
+        native.local_repository,
+        name = "com_google_googletest",
+        path = "third_party/googletest",
+    )
+
+    # Load gRPC dependency
+    maybe(
+        http_archive,
+        name = "com_github_grpc_grpc",
+        sha256 = "d6277f77e0bb922d3f6f56c0f93292bb4cfabfc3c92b31ee5ccea0e100303612",
+        strip_prefix = "grpc-1.28.0",
+        urls = [
+            "https://github.com/grpc/grpc/archive/v1.28.0.tar.gz",
+        ],
+    )
+
+    # Uses older protobuf version because of
+    # https://github.com/protocolbuffers/protobuf/issues/7179
+    maybe(
+        http_archive,
+        name = "com_google_protobuf",
+        sha256 = "b679cef31102ed8beddc39ecfd6368ee311cbee6f50742f13f21be7278781821",
+        strip_prefix = "protobuf-3.11.2",
+        urls = [
+            "https://github.com/protocolbuffers/protobuf/releases/download/v3.11.2/protobuf-all-3.11.2.tar.gz",
+        ],
+    )
+
+    # OTLP Protocol definition
+    maybe(
+        native.new_local_repository,
```

Comment:
This seems coming from the `WORKSPACE` file, but still wondering why OTLP protocol definition is `native.new_local_repository` but others are `native.local_repository`?

Reply:
That's the difference between something that HAS a bazel-build already, or one where we're adapting it to be a BAZEL workspace.

OTLP does NOT have a bazel build, so we're provding our own build file.